tests/api.c

```diff
@@ -36045,6 +36049,15 @@ static void test_wolfSSL_SESSION(void)
     AssertIntEQ(wolfSSL_SESSION_is_resumable(NULL), 0);
     AssertIntEQ(wolfSSL_SESSION_is_resumable(sess), 1);
     #endif
+
+    AssertIntEQ(wolfSSL_SESSION_has_ticket(NULL), 0);
+    AssertIntEQ(wolfSSL_SESSION_get_ticket_lifetime_hint(NULL), 0);
+    AssertIntGT(wolfSSL_SESSION_get_ticket_lifetime_hint(sess), 0);
```

Comment:
Is the lifetime hint a dynamic value that depends on build config? If possible I think it would be best to test for `AssertIntEQ` here.

Reply:
Good suggestions, adjusted to check against SESSION_TICKET_HINT_DEFAULT.